plugins/reach/services/ReachProfileService.php

```diff
@@ -107,15 +107,18 @@ public function updateAction($id, KalturaReachProfile $reachProfile)
 			throw new KalturaAPIException(KalturaReachErrors::REACH_PROFILE_NOT_FOUND, $id);
 
 		// save the object
-		$dbReachProfile = $reachProfile->toUpdatableObject($dbReachProfile);
-		$credit = $dbReachProfile->getCredit();
-		if ($credit)
+		$currentCredit = $dbReachProfile->getCredit();
+		$updatedCredit = $reachProfile->credit;
+		
+		if ($updatedCredit)
 		{
-			if ($credit instanceof kReoccurringVendorCredit)
+			if ($currentCredit instanceof kReoccurringVendorCredit && $this->wasCreditUpdated($currentCredit, $updatedCredit))
```

Comment:
Move this function to KalturaRoccuringVendorCredit and have it check values vs $this->